sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1227,80 +1046,44 @@ def hpat_pandas_stringmethods_islower_impl(self):
 
 @overload_method(StringMethodsType, 'isalnum')
 def hpat_pandas_stringmethods_isalnum(self):
-    """
-        Intel Scalable Dataframe Compiler User Guide
-        ********************************************
-        Pandas API: pandas.Series.str.isalnum
-
-        Limitations
-        -----------
-        Series elements are expected to be Unicode strings. Elements cannot be NaN.
 
-        Examples
-        --------
-        .. literalinclude:: ../../../examples/series/str/series_str_isalnum.py
-           :language: python
-           :lines: 27-
-           :caption: Check if all the characters in the text are alphanumeric
-           :name: ex_series_str_isalnum
+    ty_checker = TypeChecker('Method isalnum().')
+    ty_checker.check(self, StringMethodsType)
 
-        .. command-output:: python ./series/str/series_str_isalnum.py
-            :cwd: ../../../examples
+    def hpat_pandas_stringmethods_isalnum_impl(self):
+        item_count = len(self._data)
+        result = numpy.empty(item_count, numba.types.boolean)
+        for idx, item in enumerate(self._data._data):
+            result[idx] = item.isalnum()
 
-        .. seealso::
-            :ref:`Series.str.isalpha <pandas.Series.str.isalpha>`
-                Check whether all characters are alphabetic.
-            :ref:`Series.str.isnumeric <pandas.Series.str.isnumeric>`
-                Check whether all characters are numeric.
-            :ref:`Series.str.isalnum <pandas.Series.str.isalnum>`
-                Check whether all characters are alphanumeric.
-            :ref:`Series.str.isdigit <pandas.Series.str.isdigit>`
-                Check whether all characters are digits.
-            :ref:`Series.str.isdecimal <pandas.Series.str.isdecimal>`
-                Check whether all characters are decimal.
-            :ref:`Series.str.isspace <pandas.Series.str.isspace>`
-                Check whether all characters are whitespace.
-            :ref:`Series.str.islower <pandas.Series.str.islower>`
-                Check whether all characters are lowercase.
-            :ref:`Series.str.isupper <pandas.Series.str.isupper>`
-                Check whether all characters are uppercase.
-            :ref:`Series.str.istitle <pandas.Series.str.istitle>`
-                Check whether all characters are titlecase.
+        return pandas.Series(result, self._data._index, name=self._data._name)
 
-        Intel Scalable Dataframe Compiler Developer Guide
-        *************************************************
+    return hpat_pandas_stringmethods_isalnum_impl
 
-        Pandas Series method :meth:`pandas.core.strings.StringMethods.isalnum()` implementation.
 
-        Note: Unicode type of list elements are supported only. Numpy.NaN is not supported as elements.
+hpat_pandas_stringmethods_istitle.__doc__ = hpat_pandas_series_str_is_docstring_tmpl.format(**{
```

Comment:
I think it's better to do this in a loop, iterating over keys, values of a dict like:
stringmethods_funcs = {
    'istitle':  hpat_pandas_stringmethods_istitle,
     ...
}